src/abi_llvm.cpp

```diff
@@ -0,0 +1,38 @@
+//===-- abi_llvm.cpp - LLVM Target ABI description --------------*- C++ -*-===//
+//
+//                         LDC – the LLVM D compiler
+//
+// This file is distributed under the BSD-style LDC license. See the LICENSE
```

Comment:
this was copied verbatim from the LDC source file. you're probably right that we should update it to contain the relevant text from their LICENSE file: https://github.com/ldc-developers/ldc/blob/908815256ffe72b04c76309aa0cdbd439f81faed/LICENSE#L32-L58